packages/suite-desktop/src-electron/libs/http-receiver.ts

```diff
@@ -56,6 +57,7 @@ export class HttpReceiver extends EventEmitter {
              */
         ];
         this.server = http.createServer(this.onRequest);
+        this.logger = global.logger;
```

Comment:
```suggestion
       const { logger } = global;
```

just a nitpick, but wouldn't be better to have logger in upper scope instead of class scope?
is it relevant for file to have exposed logger?

Reply:
I prefer to have it as a property of a class but I will mark it as private, good spot!

Reply:
Addressed in  417a0f7.